IPs/controller_1.0/drivers/controller_v1_0/src/controller.c

```diff
@@ -10,9 +10,16 @@ void exec(UINTPTR baseAddr, u32 ins)
 
     // Write start signal to control register 0
     CONTROLLER_mWriteReg(baseAddr, 0, 1);
+#include "xparameters.h"
+    printf("Before: %lx\n", Xil_In32(XPAR_AXI_GPIO_0_BASEADDR));
 
     // Busy waiting for execution
     while (CONTROLLER_mReadReg(baseAddr, 0));
+    printf("After: %lx\n", Xil_In32(XPAR_AXI_GPIO_0_BASEADDR));
+    printf("After: %lx\n", Xil_In32(XPAR_AXI_GPIO_0_BASEADDR));
+    printf("After: %lx\n", Xil_In32(XPAR_AXI_GPIO_0_BASEADDR));
+    printf("After: %lx\n", Xil_In32(XPAR_AXI_GPIO_0_BASEADDR));
+    printf("After: %lx\n", Xil_In32(XPAR_AXI_GPIO_0_BASEADDR));

```

Comment:
I used these code to read the value from AXI_GPIO, which is added to the block design by me to read the output of DSP directly.